File: Practica2/Python/Punto.java
public class Punto {
    private float x;
    private float y;

    public Punto(float x, float y) {
        this.x = x;
        this.y = y;
    }

    public String coord_cartesianas() {
        return "(" + x + ", " + y + ")";
    }

    public String coord_polares() {
        double r = Math.sqrt(x * x + y * y);
        double theta = Math.atan2(y, x);
        return "(r=" + r + ", θ=" + Math.toDegrees(theta) + "°)";
    }

    @Override
    public String toString() {
        return "Punto  " + "x=" + x + ", y=" + y ;
    }
}

File: Practica1/punto.py
import math
class Punto:
    def __init__(self, x, y):
        self.x = x
        self.y = y
    def coord_cartesianas(self):
        return self.x, self.y
    def coord_polares(self):
        radio = math.sqrt(self.x * self.x + self.y * self.y)
        angulo = math.atan2(self.y,self.y)
        angulo = math.degrees(angulo)
        return radio, angulo
    def __str__(self):
        return "({:.2f},{:.2f})".format(self.x, self.y)
p1 = Punto(0,3)
print("Punto 1")
print(p1)
x,y = p1.coord_cartesianas()
print("Coordenadas cartesianas")
print(x,y)
r,a = p1.coord_polares()
print("Coordenadas polares")
print(r,a)